silver/models/payment_methods.py

```diff
@@ -55,6 +55,9 @@ def as_list(cls):
     verified = models.BooleanField(default=False)
     canceled = models.BooleanField(default=False)
 
+    valid_until = models.DateTimeField(null=True, blank=True)
+    display_info = models.CharField(max_length=256, null=True, blank=True)
```

Comment:
Just a small info to easily identify a payment method.
For PayPal should be the email and for credit card the last 4 digits.